flag_engine/utils/hashing.py

```diff
@@ -1,9 +1,11 @@
 import hashlib
 import typing
 
+from flag_engine.utils.types import SupportsStr
+
 
 def get_hashed_percentage_for_object_ids(
-    object_ids: typing.Iterable[typing.Any], iterations: int = 1
+    object_ids: typing.Iterable[SupportsStr], iterations: int = 1
```

Comment:
Why `SupportsStr` ?

Reply:
This is the broadest correct type I could think of without resorting to either `Any` or a `type: ignore` comment.